directory.py

```diff
@@ -0,0 +1,69 @@
+from Person import Person
+from json_reader import JSONReader
+
+class Directory:
+    providers = {}
+    l = list()
+
+    def __init__(self):
+        self.providers['airtel'] = ['9900', '9800', '9811']
+        self.providers['BSNL'] = ['9440', '9822']
+        self.providers['Idea'] = ['9848', '9912']
+        self.providers['Reliance'] = ['9300', '9812']
+        obj = JSONReader()
+        self.l = obj.get_all()
+
+
+
+    def get_filter(self, field, value):
+        for i in range(len(self.l)):
```

Comment:
Try this test case.

1. Add a contact with name "abc", phone "123".
2. Then filter the contacts with name "abc"

Is your directory class returning the Contact?